ops/pebble.py

```diff
@@ -1822,10 +1824,10 @@ def _reader_to_websocket(
             chunk = chunk.encode(encoding)
         ws.send_binary(chunk)
 
-    ws.send('{"command":"end"}')  # type: ignore # Send "end" command as TEXT frame to signal EOF
+    ws.send('{"command":"end"}')
```

Comment:
I think it's worth keeping the explanatory comment.

Reply:
Good catch, thank you!

I've fixed this in #1333 and rebased.